packages/invoices/src/types.ts

```diff
@@ -12,6 +12,7 @@ export interface DraftInboundLine extends DraftInboundLineFragment {
 export interface DraftOutboundLine extends OutboundLineFragment {
   isCreated: boolean;
   isUpdated: boolean;
+  isDeleted?: boolean;
```

Comment:
Had been using checks for number of packs and things to know when to delete/prune.

I thought service items might be better to stay with no charge. I've had invoices with `$0` 🤷

Reply:
yes, agree. different with quantities - but prices can be 0